test/Dialect/SV/hw-extract-test-code.mlir

```diff
@@ -134,24 +134,38 @@ module attributes {firrtl.extract.assert =  #hw.output_file<"dir3/", excludeFrom
 }
 
 // -----
-// Check "empty" modules are extracted
+// Check "empty" modules are inlined
 
-// CHECK-LABEL: @InputOnly(
-// CHECK-SAME: output_file = #hw.output_file<"testbench/", excludeFromFileList, includeReplicatedOps>
-// CHECK: hw.instance "input_only" sym @[[sym_name:[^ ]+]] @InputOnly
-// CHECK: sv.bind <@Top::@[[sym_name]]> {output_file = #hw.output_file<"cover/bind.sv", excludeFromFileList, includeReplicatedOps>}
-module attributes {
-  firrtl.extract.cover.bindfile = #hw.output_file<"cover/bind.sv", excludeFromFileList, includeReplicatedOps>,
-  firrtl.extract.testbench = #hw.output_file<"testbench/", excludeFromFileList, includeReplicatedOps>
-} {
-  hw.module @InputOnly(%clock: i1, %cond: i1) -> () {
+// CHECK-NOT: @InputOnly(
+// CHECK-DAG: @InputOnly_assert(
+// CHECK-DAG: @InputOnly_cover(
+// CHECK-DAG: @InputOnlySym_cover(
+// CHECK-LABEL: @InputOnlySym(
+// CHECK: hw.instance "{{[^ ]+}}" sym @[[input_only_sym_cover:[^ ]+]] @InputOnlySym_cover
+// CHECK-LABEL: @Top
+// CHECK: hw.instance "{{[^ ]+}}" sym @[[input_only_assert:[^ ]+]] @InputOnly_assert
+// CHECK: hw.instance "{{[^ ]+}}" sym @[[input_only_cover:[^ ]+]] @InputOnly_cover
+// CHECK: hw.instance "{{[^ ]+}}" {{.+}} @InputOnlySym
+// CHECK-DAG: sv.bind <@Top::@[[input_only_assert]]>
+// CHECK-DAG: sv.bind <@Top::@[[input_only_cover]]>
+// CHECK-DAG: sv.bind <@InputOnlySym::@[[input_only_sym_cover]]>
```

Comment:
The PR may want to include a `CHECK-NOT` on the original modules. This looks good otherwise.